extension-manifest-v3/src/background/stats.js

```diff
@@ -318,7 +326,16 @@ if (__PLATFORM__ !== 'safari' && __PLATFORM__ !== 'firefox') {
       if (details.error !== 'net::ERR_BLOCKED_BY_CLIENT') return;
 
       const stats = tabStats.get(details.tabId);
-      stats?.requestsBlocked.push(details.requestId);
+      if (!stats) return;
```

Comment:
`stats.requestsBlocked` was not used anywhere. I think it might be related to anti-tracking feature changes. Instead of passing those ids to the array, requests should be marked.

Reply:
requestsBlocked are for the ad-blocking. anti-tracking marks requests as modified